### PR TITLE
Add unimplemented method to java.util.Properties

### PR DESCRIPTION
**Pull request by ekrich**
This is an extra method needed to link `ekrich/sconfig` that was found when working on the `java.net.URL` code change but it seems better as a separate PR. We have had some PRs working on `Properties` but nothing solid yet and this should be easy to rebase for WIP.

EDIT: This is targeted for both `0.3.x` and `master`.

### CODE REVIEW

File: javalib/src/main/scala/java/util/Properties.scala [hunk 1]
```diff
@@ -14,6 +14,8 @@ class Properties(protected val defaults: Properties)
 
   def load(inStream: java.io.InputStream): Unit = ???
 
+  def load(reader: java.io.Reader): Unit = ???
```
Comment (densh) [inline javalib/src/main/scala/java/util/Properties.scala]: Please add `@stub` annotation.

Comment (ekrich) [reply javalib/src/main/scala/java/util/Properties.scala]: In general, are `@stub` suppose to be everywhere there is an empty implementation?

Comment (densh) [reply javalib/src/main/scala/java/util/Properties.scala]: Responded on gitter: :point_up: [March 21, 2019 12:20 PM](https://gitter.im/scala-native/scala-native?at=5c9373675547f774483ec225)

**Comment by densh** [comment]
LGTM, thanks @ekrich ! 👍

**Comment by ekrich** [comment]
Refer to Issue #1412

**Pull request merged**